ztp/source-crs/DisconnectedICSP.yaml

```diff
@@ -5,5 +5,6 @@ metadata:
     annotations:
         ran.openshift.io/ztp-deploy-wave: "1"
 spec:
-    repositoryDigestMirrors: $mirrors
+    repositoryDigestMirrors:
+    - mirrors
```

Comment:
Same note on commenting this out instead.

Reply:
yes, done